pkg/kubectl/cmd/delete_test.go

```diff
@@ -70,7 +70,7 @@ func TestDeleteNamedObject(t *testing.T) {
 		Codec: codec,
 		Client: fake.CreateHTTPClient(func(req *http.Request) (*http.Response, error) {
 			switch p, m := req.URL.Path, req.Method; {
-			case p == "/namespaces/test/replicationcontrollers/redis-master-controller" && m == "DELETE":
+			case p == "/namespaces/test/replicationcontrollers/redis-master-controller" && (m == "DELETE" || m == "GET"):
 				return &http.Response{StatusCode: 200, Body: objBody(codec, &rc.Items[0])}, nil
 			default:
 				// Ensures no GET is performed when deleting by name
```

Comment:
these tests were explicitly here to make sure deletes succeed when GET fails (the case where a user is allowed to DELETE something they cannot GET)... we still need to test those scenarios.